coconut/cmd/configuration_import.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * === This file is part of ALICE O² ===
+ *
+ * Copyright 2019 CERN and copyright holders of ALICE O².
+ * Author: George Raduta <george.raduta@cern.ch>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * In applying this license CERN does not waive the privileges and
+ * immunities granted to it by virtue of its status as an
+ * Intergovernmental Organization or submit itself to any jurisdiction.
+ */
+
+package cmd
+
+import (
+	"github.com/AliceO2Group/Control/coconut/configuration"
+	"github.com/spf13/cobra"
+)
+
+var configurationImportCmd = &cobra.Command{
+	Use:   "import <component> <entry> <file_path>",
+	Aliases: []string{"i", "imp"},
+	Example: `coconut conf import <component> <entry> <file_path>
+coconut conf import <component/entry> <file_path>
```

Comment:
Minor nitpick: I'd put `<component>/<entry>` instead of `<component/entry>` as `component` and `entry` are the variables here, which we compose in one way or another.